ios/Podfile.lock

```diff
@@ -8,7 +8,7 @@ PODS:
   - FMDB/standard (2.7.5)
   - image_cropper (0.0.4):
     - Flutter
-    - TOCropViewController (~> 2.6.1)
+    - TOCropViewController (~> 2.7.4)
```

Comment:
1. This file must not be included in your PR
2. Please exlcude it

Reply:
Hey Palisadoes, if I do not update the `TOCropViewController` version from `2.6.1` to `2.7.4` in `Podfile.lock` then The build for iOS fails as pod checks the snapshot from `Podfile.lock` and `Podfile`